File: Producto2/juego.java
public class juego{
    public static void main(String[]args){
	System.out.println("Hola!, Tratare de adivinar un numero");


	try{
	    Thread.sleep(5000);
	}

	catch(InterruptedException ex){
	    Thread.currentThread().interrupt();
	}

	System.out.println("Multiplicalo por 9");

	try{
	    Thread.sleep(5000);
	}

	catch(InterruptedException ex){
	    Thread.currentThread().interrupt();
	}

	System.out.println("Si el numero es de dos digitos, sumalos entre si.");

	try{
	    Thread.sleep(5000);
	}

	catch(InterruptedException ex){
	    Thread.currentThread().interrupt();
	}

	System.out.println("Al numero resultante sumale 4 ");
	try{
	    Thread.sleep(10000);
	}

	catch(InterruptedException ex){
	    Thread.currentThread().interrupt();
	}
	
	System.out.println("Muy bien. El resultado es 13 :)");
    }
}

File: Producto2/juego.c
#include <stdio.h>
#include <unistd.h> 
int main()
{
    printf("Hola! Trataré de adivinar un número.\n");

 printf("Piensa un número entre 1 y 10.\n");
    sleep(5); 
    printf("Ahora multiplícalo por 9.\n");
    sleep(5); 
    printf("Si el número tiene 2 dígitos, súmalos entre si: Ej. 36 -> 3+6=9. Si tu número tiene un solo dígito, súmale 0.\n");
    sleep(5); 
    printf("Al número resultante súmale 4.\n");
    sleep(10); 
    printf("Muy bien. El resultado es 13 :) \n");


}
